Formula/openssl@1.1.rb

```diff
@@ -4,9 +4,6 @@ class OpensslAT11 < Formula
   url "https://www.openssl.org/source/openssl-1.1.1s.tar.gz"
   mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.1.1s.tar.gz"
   mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.1.1s.tar.gz"
-  mirror "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1s.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/old/1.1.1/openssl-1.1.1s.tar.gz"
-  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/old/1.1.1/openssl-1.1.1s.tar.gz"
```

Comment:
Please don't remove these. These mirrors go live when the earlier URLs start failing.

Reply:
Reverting changes